kzz7yh-a04629/cod-ve07v1_kzz7yh-a04629.xml

```diff
@@ -388,6 +391,7 @@
       <lb ed="#V" n="41"/>plus tendit in principium motorem quam in motores secundarios: ex quod 
       <lb ed="#V" n="42"/>sequitur quod cum voluntas moueatur ab ipso deo sicut a primo 
       <lb ed="#V" n="43"/>motore non potest acquietari plenarie in ipso. 
+      <!--missing paragraph with Ad Primum-->
```

Comment:
Lines 44 and 45 are the lines for this break; the text just seems to be garbled and needs to be corrected.
```suggestion
      <!--missing paragraph with Ad Primum-->
```